pkg/server/middleware.go

```diff
@@ -23,7 +25,7 @@ func authMiddlewareWithUser(a *auth.Authenticator, handlerFunc func(user *auth.U
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		user, err := a.Authenticate(r)
 		if err != nil {
-			plog.Infof("authentication failed: %v", err)
+			klog.V(2).Infof("authentication failed: %v", err)
```

Comment:
Out of curiosity, why the verbosity is set to 2 here?

Reply:
Not positive, I think prior discussions (with people who have left) said just default to v(2) for info messages.  I am fine removing the v(2).

Reply:
yeah, I would remove it since we are not using it anywhere else in the `server`pkg 👍